rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc

```diff
@@ -0,0 +1,57 @@
+= Search in Google Drive
+
+Search for files or folders in your Google Drive, including your shared drives.
+
+You can add variables, filters, and operations to your search query. This action step returns the number of matches found, the file or folder IDs, and the names of the files or folders found. 
+
+== Properties 
+
+include::partial$oauth-authentication-settings.adoc[]
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *Include all files from shared drives* 
++
+Select this option to search files and folders in your shared drives in addition to My Drive. 
+** *Search Query*
++
+String containing the search query. Use the Wizard for assistance while building your query. 
```

Comment:
Maybe add that that, additional to the provided ones, all filters are supported, and users can look them up here https://developers.google.com/drive/api/guides/ref-search-terms